### PR TITLE
Update administrativeunit-post-scopedrolemembers.md

### PR DESCRIPTION
**Pull request by VinceSmith**

### CODE REVIEW

**Comment by VinceSmith** [comment]
#sign-off

File: api-reference/beta/api/administrativeunit-post-scopedrolemembers.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ One of the following permissions is required to call this API. To learn more, in
 
 |Permission type      | Permissions (from least to most privileged)              |
 |:--------------------|:---------------------------------------------------------|
-|Delegated (work or school account) | Directory.AccessAsUser.All    |
+|Delegated (work or school account) | AdministrativeUnit.ReadWrite.All, Directory.AccessAsUser.All    |
```
Comment (Lauragra) [inline api-reference/beta/api/administrativeunit-post-scopedrolemembers.md]: @VinceSmith , @Jackson-Woods , this permission isn't currently documented in the Permissions reference. Can we update that topic as part of this PR? Thanks.

Comment (VinceSmith) [reply api-reference/beta/api/administrativeunit-post-scopedrolemembers.md]: It's actually updated already, in a different PR. I was heads down making updates out of main and then when I started adding #sign-off I realized all of the separate PRs is going to be a pain for reviewers. Sorry about that.

Comment (VinceSmith) [reply api-reference/beta/api/administrativeunit-post-scopedrolemembers.md]: https://github.com/microsoftgraph/microsoft-graph-docs/pull/5224

**Pull request closed without merging**